WooCommerce/Classes/POS/Presentation/ItemCardView.swift

```diff
@@ -3,13 +3,16 @@ import SwiftUI
 
 struct ItemCardView: View {
     private let item: POSItem
-    private let onItemCardTapped: (() -> Void)?
 
     @ScaledMetric private var scale: CGFloat = 1.0
 
-    init(item: POSItem, onItemCardTapped: (() -> Void)? = nil) {
+    init(item: POSItem) {
         self.item = item
-        self.onItemCardTapped = onItemCardTapped
+    }
+
+    var commaSeparatedItemCategories: String {
+        let maxNumberOfCategories = 3
+        return item.itemCategories.prefix(maxNumberOfCategories).joined(separator: ", ")
```

Comment:
<img width="429" alt="Screenshot 2024-06-17 at 17 58 32" src="https://github.com/woocommerce/woocommerce-ios/assets/6242034/acd52c49-9bba-469a-bf37-bba6bacfa2c0">

what do you think about maybe ignoring this "Uncategorized" category? I remember we were talking about it during one of the sync calls but I do not remember what we decided.

Reply:
Yeah, that was my initial approach, but since the category comes as a String it changes depending on the store language, we need to ignore this string when translated to all languages:

> I found that choosing the first on the array generally returns the default "Uncategorized" unless has been specifically unselected from the product. We could filter this one out, but then we have to deal with i18n as well when validating, so for now it feels good enough for testing.

I'm not quite sure of the best solution for it (or if it's even a problem!), but I'll log it to follow-up and see how we decide to tackle it.